frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu.dart

```diff
@@ -121,17 +175,24 @@ class HomeMenu extends StatelessWidget {
               return ReorderableListView.builder(
                 itemCount: menuItems.length,
                 buildDefaultDragHandles: false,
-                header: Padding(
-                  padding:
-                      EdgeInsets.only(bottom: 20.0 - MenuAppSizes.appVPadding),
-                  child: MenuUser(user),
+                header: Column(
+                  children: [
+                    Padding(
+                      padding: EdgeInsets.only(
+                        bottom: 20.0 - MenuAppSizes.appVPadding,
+                      ),
+                      child: MenuUser(user),
+                    ),
```

Comment:
I know you didn't add this, but this seems quite weird. `20.0` is a magic number, and subtracting a constant doesn't make it less magic... 
```suggestion
                    Padding(
                      padding: EdgeInsets.only(bottom: 6),
                      child: MenuUser(user),
                    ),
```